database/migrations/2024_08_10_000000_create_game_set_links_table.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::create('game_set_links', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('parent_game_set_id');
+            $table->unsignedBigInteger('child_game_set_id');
+            $table->timestamps();
+        });
+
+        Schema::table('game_set_links', function (Blueprint $table) {
+            $table->foreign('parent_game_set_id')
+                ->references('id')
+                ->on('game_sets')
+                ->onDelete('cascade');
+
+            $table->foreign('child_game_set_id')
+                ->references('id')
+                ->on('game_sets')
+                ->onDelete('cascade');
+
+            $table->index('parent_game_set_id');
+            $table->index('child_game_set_id');
+        });
+
+        Schema::table('game_sets', function (Blueprint $table) {
+            $table->string('type')->after('id');
+            $table->string('title', 80)->nullable()->after('type');
+            $table->renameColumn('legacy_game_id', 'game_id');
+        });
```

Comment:
There are two types of `game_sets`: "hub" and "game" (see _GameSetType.php_).

"hub" is self-explanatory.
"game" is to facilitate "Similar Games" functionality.

If hubs are to be broken away from `GameData`, they need titles. Thus, a `title` column has been added to this table.
Additionally, `legacy_game_id` has been renamed to `game_id`. This is more appropriate for facilitating the "Similar Games" functionality. `game_id` will still be used in the future for hub game page redirects (/game/12345 -> /hub/12345).